README.md

```diff
@@ -206,6 +206,44 @@ builder.Services.AddForteEpiResponsivePicture();
 
 If you want to have custom logic of creating `<picture>` or `<source>` elements you can register your own implementations of `ISourceTagBuilderProvider` and/or `IPictureTagBuilderProvider` which can produce your own builders. 
 
+## How to start using React
+Apart from nuget there is npm package published which contains React component which can be used as a equivalent for `<picture>` element byt for React. The details provided above are still valid form the conceptual perspective. Below code snippet for React can be found:
```

Comment:
`byt` instead of `but`

Reply:
should we mention the name of npm package?

Reply:
done